src/uct/ib/base/ib_log.c

```diff
@@ -154,7 +157,7 @@ static void uct_ib_dump_wr(struct ibv_qp *qp, uct_ib_opcode_t *op,
 }
 
 static void uct_ib_dump_send_wr(uct_ib_iface_t *iface, struct ibv_qp *qp,
-                                struct ibv_send_wr *wr,
+                                struct ibv_send_wr *wr, int max_log_sge,
```

Comment:
why need to pass this as arg, iface already points to ib_device

Reply:
Just wondered if there is any case where want to dump  >1 SGEs in non-zcopy am cases

Reply:
removed arg  passing